--- origsrc/gtkam-0.1.15/src/Makefile.am	2008-01-03 07:42:29.000000000 -0600
+++ src/gtkam-0.1.15/src/Makefile.am	2008-04-14 16:25:12.828125000 -0500
@@ -54,7 +54,7 @@
 	$(GTKAM_LIBS) 	\
 	$(BONOBO_LIBS)	\
 	$(GNOME_LIBS)	\
-	$(INTLLIBS)
+	$(LTLIBINTL)
 
 if HAVE_GIMP
 gtkam_gimp_libexecdir = $(libdir)/gimp/$(GIMP_VERSION)/plug-ins
@@ -64,12 +64,12 @@
 	$(MY_CPPFLAGS)		\
 	$(GIMP_CFLAGS)
 gtkam_gimp_LDADD = \
+	libgtkam.a		\
 	$(GIMP_LIBS)	\
 	$(LIBEXIF_LIBS)		\
 	$(LIBEXIF_GTK_LIBS) \
 	$(LIBGPHOTO2_LIBS)	\
 	$(BONOBO_LIBS)		\
 	$(GNOME_LIBS)		\
-	$(INTLLIBS)		\
-	libgtkam.a
+	$(LTLIBINTL)	
 endif
--- origsrc/gtkam-0.1.15/test/Makefile.am	2006-08-27 14:01:24.000000000 -0500
+++ src/gtkam-0.1.15/test/Makefile.am	2008-04-14 16:25:12.921875000 -0500
@@ -9,13 +9,13 @@
 	$(GTKAM_GIMP_CFLAGS)
 
 MY_LIBS = \
+	$(top_builddir)/src/libgtkam.a	\
 	$(LIBEXIF_LIBS)			\
 	$(LIBEXIF_GTK_LIBS)		\
 	$(LIBGPHOTO2_LIBS)	\
 	$(GTK_LIBS)			\
 	$(BONOBO_LIBS)			\
-	$(top_builddir)/src/libgtkam.a	\
-	$(INTLLIBS)
+	$(LTLIBINTL)
 
 noinst_PROGRAMS = test-fsel test-cancel
 
