--- origsrc/gtkam-0.1.15/Makefile.am	2008-01-03 07:42:19.000000000 -0600
+++ src/gtkam-0.1.15/Makefile.am	2008-04-14 16:25:12.656250000 -0500
@@ -12,8 +12,7 @@
 ########################################################################
 # Install a few support and doc files
 
-gtkamdocdir = $(datadir)/doc/gtkam
-gtkamdoc_DATA = AUTHORS COPYING NEWS README TODO CHANGES
+noinst_DATA = AUTHORS COPYING NEWS README TODO CHANGES
 
 desktopdir = $(datadir)/applications
 desktop_DATA = gtkam.desktop
